scripts/mirna_extension.py

```diff
@@ -189,17 +186,16 @@ def main(args):
             return
 
     # Create dictionary with the ref. sequence length
+    seq_lengths: Optional[Dict] = None
     if args.chr:
         seq_lengths = {}
         with open(args.chr, 'r') as f:
             for line in f:
                 ref_seq, length = line.strip().split("\t")
                 seq_lengths[ref_seq] = int(length)
-    else:
-        seq_lengths = None
 
     m = MirnaExtension()
-    m.load_gff_file(args.input)
+    m.load_gff_file(str(args.input))
```

Comment:
I'd use type `Optional[Path]` in `load_gff_file()` as well. If `gffutils.create_db()` cannot deal with `pathlib.Path`, then you can still convert with `str()`.